configure.ac

```diff
@@ -330,6 +330,21 @@ if test "$my_htop_platform" = darwin; then
    AC_CHECK_FUNCS([mach_timebase_info])
    AC_CHECK_DECLS([IOMainPort], [], [], [[#include <IOKit/IOKitLib.h>]])
    AC_CHECK_DECLS([IOMasterPort], [], [], [[#include <IOKit/IOKitLib.h>]])
+
+   AC_MSG_CHECKING([for vm_statistics64 supported])
```

Comment:
I thought you can use `AC_CHECK_TYPES` macro to simplify the check.
Maybe this is sufficient:
```shell
AC_CHECK_TYPES([struct vm_statistics64], [], [], [[#include <mach/vm_statistics.h>]]
dnl This defines HAVE_STRUCT_VM_STATISTICS64
```